### ISSUE TITLE
Create simple mongodb example with specific version of node & npm

### ISSUE THREAD

**Issue by kenyk7** [arackaf/mongo-graphql-starter#3]

**Comment by arackaf** [comment]
Sorry?  Not sure what this issue it looking for?  A blog post about how to use the utility?

**Comment by kenyk7** [comment]
Exactly, but I already made a demonstration and I will be testing with the new updates of this library:
Example:

https://github.com/kenyk7/graphcool-local
